udata_front/theme/gouvfr/assets/less/components/buttons.less

```diff
@@ -87,7 +87,7 @@ These are now removed, see tags instead.
     border-radius: 3em;
 }
 
-.fr-btn--sm {
+.fr-btns-group--sm .fr-btn:not([class^=fr-icon-]):not([class*=" fr-icon-"]):not([class^=fr-fi-]):not([class*=" fr-fi-"]), .fr-btn--sm {
```

Comment:
Oh wow, do we need to go with this complex selector?

Reply:
This is the DSFR selector that we are overwritting so I'm using it as is. We can narrow it or use a custom logic but it won't be a perfect match anymore

Reply:
Nah let's keep it! It's just that I'm not completely fluent in CSS selector language ahah

Reply:
The syntax is quite odd, I agree. It targets `.fr-btn` that have no icon related classes. `class^` matches a class attribute starting with `fr-icon-`  and `class*` matches a class attribute that contains ` fr-icon-`.